app/core/utils.js

```diff
@@ -2135,6 +2135,7 @@ const freeAccessLevels = [
 const orgKindString = function (kind, org = null) {
   if ((kind === 'administrative-region') && ((org != null ? org.country : undefined) === 'US') && /^en/.test(me.get('preferredLanguage'))) { return 'State' }
   const key = {
+    'administrative-country': 'outcomes.country',
```

Comment:
We can call this just `'country'`, we don't need the `'administrative-'` prefix ("administrative region" is a general term for a state, province, canton, oblast, etc.